gems/pending/util/postgres_admin.rb

```diff
@@ -53,6 +53,20 @@ def self.database_disk_filesystem
     "xfs".freeze
   end
 
+  def self.initialized?
+    !Dir[data_directory.join("*")].empty?
+  end
+
+  def self.service_running?
+    LinuxAdmin::Service.new(service_name).running?
+  end
+
+  def self.local_server_status
+    return "running" if service_running?
+    return "initialized and stopped" if initialized?
+    "not initialized"
```

Comment:
In this case, I think I like this more: (if we were to add more conditions later, this should become a simple PG server status class.)

```ruby
if service_running?
  "running" 
elsif initialized?
  "initialized and stopped"
else
  "not initialized
end
```